cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -122,7 +122,6 @@ start_etcd() {
   ${ETCD_CMD} \
     --name="etcd-$(hostname)" \
     --debug \
-    --force-new-cluster \
```

Comment:
Yes - this is not used for recovers. To recover from backup we are using:
https://github.com/kubernetes/kubernetes/blob/master/cluster/restore-from-backup.sh